lib/Runtime/Library/ModuleRoot.cpp

```diff
@@ -406,29 +406,29 @@ namespace Js
         return this->GetLibrary()->GetGlobalObject()->GlobalObject::DeleteRootProperty(propertyId, flags);
     }
 
-    BOOL ModuleRoot::HasItem(uint32 index)
+    PropertyQueryFlags ModuleRoot::HasItemQuery(uint32 index)
     {
-        return DynamicObject::HasItem(index)
-            || (this->hostObject && JavascriptOperators::HasItem(this->hostObject, index));
+        return JavascriptConversion::BooleanToPropertyQueryFlags(JavascriptConversion::PropertyQueryFlagsToBoolean((DynamicObject::HasItemQuery(index)))
```

Comment:
Indeed - thanks :)

---
In reply to: [106304536](https://github.com/Microsoft/ChakraCore/pull/2640#discussion_r106304536) [](ancestors = 106304536)